guide/glossary.md

```diff
@@ -111,13 +111,24 @@ Differentiate between
 
 Bitcoin with a capital *B* is typically associated with Bitcoin the protocol and payment network. It is also often used to refer to as the ecosystem as a whole when writing about it in general terms. Bitcoin with a lowercase “b” written as “bitcoin” is usually associated specifically with bitcoin as the currency.
 
-### Bitcoin Client
+### Bitcoin client
 
--What is considered a Bitcoin client? I'd suggest it's a software package comprising node / base layer wallet features. [Several other](https://coin.dance/nodes) clients exist besides Bitcoin Core. -Bosch
+Software that runs and/or connects to the Bitcoin network.
+
+### Bitcoin Core client
+
+Software considered the reference implementation for the Bitcoin protocol. It is the continuation of Satoshi Nakamoto's original Bitcoin client released 9th January, 2009. 
```

Comment:
Can also state here that the bitcoin core client is full node capable, and offers a command-line interface as well as a GUI. incorporating some of what Jamaal presented about how we talk about core.

Reply:
Not sure if this extra detail is necessary. People can dive into the references to discover this information.